saleor/payment/migrations/0034_payment_partial.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.5 on 2021-09-15 10:37
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("payment", "0033_payment_complete_order"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="payment",
+            name="partial",
+            field=models.BooleanField(default=False),
+        ),
+    ]
```

Comment:
You can merge this migration with the previous one.

Reply:
Let's leave this out until the whole epic is done. This entire feature branch will have to be eventually rebased to 3.0, so I'm expecting a full migration history rewrite prior to making a PR to master / 3.0.